### PR TITLE
Feature/tgr 31 32 40 45

### PR DESCRIPTION
**Pull request by Chugaister**
This pull request coveres few tickets (their numbers in branch name)

Conversation status bug fixed (I hope so)
Added Conversation.timestamp and Conversation.group fields
STT improved (lower timeouts, better stt model, considering GB English)

This PR is supposed to fix unexpected twilio's hangups

### CODE REVIEW

File: utils/twcustom.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+
+from twilio.twiml.voice_response import VoiceResponse
+
+
+class CustomVoiceResponse(VoiceResponse):
+    voice = "Polly.Arthur-Neural"
+
+    # gather parameters
+    timeout = 3
+    language = "en-GB"
```
Comment (Ap3lsin4k) [inline utils/twcustom.py:10]: oh yeah baby, that's the one we need.

File: utils/twcustom.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
+
+from twilio.twiml.voice_response import VoiceResponse
+
+
+class CustomVoiceResponse(VoiceResponse):
+    voice = "Polly.Arthur-Neural"
```
Comment (Ap3lsin4k) [inline utils/twcustom.py]: excuse me, 
Can we use a young female British voice?

image

File: api/twilioapi/twilioapi.py [hunk 3]
```diff
@@ -137,7 +139,7 @@ async def get_partial_result(
 async def stream_ai_response(ai_response):
     UnstableSpeechResultList = []
     for word in ai_response.split():
-        await sleep(0.2)
+        await sleep(0.5)
```
Comment (Ap3lsin4k) [inline api/twilioapi/twilioapi.py:142]: is this how long it waits before generating response? 
0.5 seconds? What is the lowest that we can set for testing?

**Review by Ap3lsin4k — COMMENTED** [review]
After tweaking the voice model and hyperparameters I am happy to merge it.

It is late, my brain is not raining, maybe I could see more nitpicks tomorrow.

File: utils/twcustom.py (hunk 2 above)
Comment (domiNo39) [reply utils/twcustom.py]: done

File: api/twilioapi/twilioapi.py (hunk 3 above)
Comment (domiNo39) [reply api/twilioapi/twilioapi.py:142]: it's delay between transcribed speech sent via sockets for live speech transcript. 0.5 is the best value

**Pull request merged**